package.json

```diff
@@ -23,7 +23,7 @@
     "@babel/cli": "^7.8.4",
     "@babel/core": "^7.9.0",
     "@babel/preset-env": "^7.9.5",
-    "jest": "^25.5.4"
+    "jest": "^29.0.0"
```

Comment:
Tests did not run successfully locally under Node 20 until Jest was updated, something about `node:events`